assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.stories.js

```diff
@@ -19,16 +19,97 @@
 /**
  * Internal dependencies
  */
+import { provideModuleRegistrations } from '../../../../../../tests/js/utils';
+import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
 import SettingsEdit from './SettingsEdit';
+import { publications } from '../../datastore/__fixtures__';
+import {
+	MODULES_READER_REVENUE_MANAGER,
+	MODULE_SLUG,
+} from '../../datastore/constants';
+import { enabledFeatures } from '../../../../features';
 
 function Template() {
 	return <SettingsEdit />;
 }
 
 export const Default = Template.bind( {} );
 Default.storyName = 'Default';
+Default.scenario = {};
+
+export const PublicationSelected = Template.bind( {} );
+PublicationSelected.storyName = 'PublicationSelected';
+PublicationSelected.scenario = {};
+PublicationSelected.args = {
+	setupRegistry: ( registry ) => {
+		const publication = publications[ 0 ];
+		registry
+			.dispatch( MODULES_READER_REVENUE_MANAGER )
+			// eslint-disable-next-line sitekit/acronym-case
+			.setPublicationID( publication.publicationId );
+	},
+};
+
+export const PublicationSelectedWithOnboardingStateNotice = Template.bind( {} );
+PublicationSelectedWithOnboardingStateNotice.storyName =
+	'PublicationSelectedWithOnboardingStateNotice';
+PublicationSelectedWithOnboardingStateNotice.scenario = {};
+PublicationSelectedWithOnboardingStateNotice.args = {
+	setupRegistry: ( registry ) => {
+		const publication = publications[ 1 ];
+		registry
+			.dispatch( MODULES_READER_REVENUE_MANAGER )
+			// eslint-disable-next-line sitekit/acronym-case
+			.setPublicationID( publication.publicationId );
+		registry
+			.dispatch( MODULES_READER_REVENUE_MANAGER )
+			// eslint-disable-next-line sitekit/acronym-case
+			.setPublicationOnboardingState( publication.onboardingState );
+	},
+};
 
 export default {
-	title: 'Modules/ReaderRevenueManager/Settings/SettingsEdit',
-	component: SettingsEdit,
+	title: 'Modules/ReaderRevenueManager/Components/Settings/SettingsEdit',
+	decorators: [
+		( Story, { args } ) => {
+			const setupRegistry = ( registry ) => {
+				enabledFeatures.add( 'rrmModule' );
```

Comment:
Sorry @techanvil , but this does not seem to work as intended because `WithRegistrySetup` has signature 

```js
WithRegistrySetup( { func, children } ) {
...

}
```

which does not seem to receive and do anything with `features` prop.

Reply:
Oh, yes, you're right. Sorry about that. We actually have this in a few places, which are obviously incorrect and we should fix. E.g.

https://github.com/google/site-kit-wp/blob/e112780ccddc087e03acf9fe1452120cdf3a045f/assets/js/modules/ads/components/settings/SettingsEdit.stories.js#L114-L117

https://github.com/google/site-kit-wp/blob/e112780ccddc087e03acf9fe1452120cdf3a045f/assets/js/components/conversion-tracking/ConversionTrackingToggle.stories.js#L62-L66

Looking into it, the reason those stories are still working as expected is because the features are set in the `parameters` for the story, for example:

https://github.com/google/site-kit-wp/blob/e112780ccddc087e03acf9fe1452120cdf3a045f/assets/js/modules/ads/components/settings/SettingsEdit.stories.js#L99-L101

Providing `features` in `parameters` will automatically enable the features due to this decorator which is applied to all stories:

https://github.com/google/site-kit-wp/blob/e112780ccddc087e03acf9fe1452120cdf3a045f/.storybook/preview.js#L67-L82

What is means is that rather that my suggested fix you can instead apply the following to provide the features to all stories in this file:

```js
export default {
	title: 'Modules/ReaderRevenueManager/Components/Settings/SettingsEdit',
	parameters: {
		features: [ 'rrmModule' ],
	},
```

Could you please apply the above instead of my suggestions?